tests/unit/scripting.tcl

```diff
@@ -295,7 +295,7 @@ start_server {tags {"scripting"}} {
 
     # Temporarily disable test for external until it is stabilized, see https://github.com/valkey-io/valkey/issues/770
```

Comment:
And this comment. It was added in #773 too.

```suggestion
```

Reply:
yean, i revert it in #775